src/scripts/items/BattleItem.ts

```diff
@@ -13,9 +13,9 @@ class BattleItem extends Item {
     }
 }
 
-ItemList['xAttack']         = new BattleItem(GameConstants.BattleItemType.xAttack, '+50% Bonus to Pokémon attack', 600);
-ItemList['xClick']          = new BattleItem(GameConstants.BattleItemType.xClick, '+50% Bonus to click attack', 400);
-ItemList['Lucky_egg']       = new BattleItem(GameConstants.BattleItemType.Lucky_egg, '+50% Bonus to experience gained', 800, undefined, 'Lucky Egg');
-ItemList['Token_collector'] = new BattleItem(GameConstants.BattleItemType.Token_collector, '+50% Bonus to dungeon tokens gained', 1000, undefined, 'Token Collector');
-ItemList['Item_magnet']     = new BattleItem(GameConstants.BattleItemType.Item_magnet, '+50% Chance of gaining an extra item', 1500, undefined, 'Item Magnet');
-ItemList['Lucky_incense']   = new BattleItem(GameConstants.BattleItemType.Lucky_incense, '+50% Bonus to money gained', 2000, undefined, 'Lucky Incense');
+ItemList['xAttack'] = new BattleItem(GameConstants.BattleItemType.xAttack, '+50% Bonus to Pokémon attack for 30 seconds', 600);
+ItemList['xClick'] = new BattleItem(GameConstants.BattleItemType.xClick, '+50% Bonus to click attack for 30 seconds for 30 seconds for 30 seconds', 400);
+ItemList['Lucky_egg'] = new BattleItem(GameConstants.BattleItemType.Lucky_egg, '+50% Bonus to experience gained for 30 seconds', 800, undefined, 'Lucky Egg');
+ItemList['Token_collector'] = new BattleItem(GameConstants.BattleItemType.Token_collector, '+50% Bonus to dungeon tokens gained for 30 seconds', 1000, undefined, 'Token Collector');
+ItemList['Item_magnet'] = new BattleItem(GameConstants.BattleItemType.Item_magnet, '+50% Chance of gaining an extra item for 30 seconds', 1500, undefined, 'Item Magnet');
+ItemList['Lucky_incense'] = new BattleItem(GameConstants.BattleItemType.Lucky_incense, '+50% Bonus to money gained for 30 seconds', 2000, undefined, 'Lucky Incense');
```

Comment:
These extended descriptions feel unnecessary.

Reply:
I would agree if not for the fact it was actually a recurring comment on the Discord players weren't even aware they could stack battle items pre the quantity change. I don't think it hurts to clarify the duration since otherwise it's left to experimentation. Let's new players calculate the time they'd gain, etc. 

Whatever you prefer.

Reply:
I think it would be better to display this information somewhere else rather than on the tooltip for every battle item.

Reply:
No problem, I have another idea in that case, will revert and come back to this in a seperate PR.